nyxx_interactions/lib/src/models/LockFileCommandSync.dart

```diff
@@ -0,0 +1,74 @@
+part of nyxx_interactions;
```

Comment:
File should be moved to dir like internal/sync. I don't like structures like here

Reply:
👌🏼